java-base/versions.py

```diff
@@ -19,4 +19,8 @@
         "product": "22",
         "vector": "0.41.1",
     },
+    {
+        "product": "23",
+        "vector": "0.41.1",
+    },
```

Comment:
question: Do we want to add Java 23 already, even tho we don't need it yet.

Reply:
Oh, didn't mean to commit that, and already asked in chat...

> Regarding Java 23, do you think I should just leave it until it is really needed?
> I don't think it helps anything by adding it early, as anything that would use it later would do so via bake rather than some image in a registry.
> 
> If you think there are other cases where it needs to be available in a registry, I'm happy to add it... I can just imagine someone arguing why it has been added and I don't have a good reason.

Reply:
I have removed it, but will leave this conversation open. Maybe @sbernauer has an answer.

Reply:
Sure, works for me